src/main/java/com/google/jenkins/plugins/computeengine/InstanceConfiguration.java

```diff
@@ -132,7 +132,7 @@ public class InstanceConfiguration implements Describable<InstanceConfiguration>
     private NetworkInterfaceIpStackMode networkInterfaceIpStackMode;
 
     @Deprecated
-    private boolean externalAddress;
+    private Boolean externalAddress;
```

Comment:
I set this value to a non-primitive boolean to be able to set it to `null`.
My tests show that the conversion between primitive and non-primitive works well.